manifests/service.pp

```diff
@@ -25,5 +25,19 @@
       limits          => $limits,
       notify          => Service['qpidd'],
     }
+
+    systemd::dropin_file { 'wait-for-port.conf':
+      ensure  => bool2str($qpid::ssl, 'present', 'absent'),
+      unit    => 'qpidd.service',
+      content => template('qpid/wait-for-port.conf.erb'),
+      notify  => Service['qpidd'],
+    }
+
+    if ! defined(Package['nc']) {
+      package { 'nc':
+        ensure => installed,
+        notify => Systemd::Dropin_file['wait-for-port.conf'],
```

Comment:
It's better to set `require => Package['nc']` on the `dropin_file`. If it's defined in another module you won't get a correct dependency in the current implementation.

Probably easiest to use:
```puppet
if $qpid::ssl {
  ensure_packages(['nc'])
  Package['nc'] -> Systemd::Dropin_file['wait-for-port.conf']
}
```

Reply:
yepp!